IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRail/PropertySets/Pset_RailwayAlignmentCommon/DocPropertySet.xml

```diff
@@ -1,8 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayAlignmentCommon" UniqueId="2dde4d86-50a3-497c-8beb-e98d4d424c34" ApplicableType="IfcAlignment">
-	<Properties>
-		<DocProperty xsi:nil="true" href="CantRotationAxis_1IburP5x9EXf88H5s5Xwyq" />
-		<DocProperty xsi:nil="true" href="VerticalReferenceAxis_2z2sJ89ur2jem1qknkDlcP" />
-	</Properties>
-</DocPropertySet>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayAlignmentCommon" UniqueId="2dde4d86-50a3-497c-8beb-e98d4d424c34" ApplicableType="IfcAlignment" />
```

Comment:
This pset needs to be removed completely Jon. Alex removed it in his PR already. I guess you did not have the latest baseline.

Reply:
This has been done in a subsequent commit.